nocasedict/_nocasedict.py

```diff
@@ -141,97 +141,137 @@ def __iter__(self):
 
 
 class NocaseDict(MutableMapping):
-    # pylint: disable=line-too-long
     """
-    A case-insensitive, ordered, and case-preserving, dictionary.
+    A case-insensitive and case-preserving ordered dictionary.
 
     The dictionary is case-insensitive: Whenever items of the dictionary are
-    looked up by key or key values are compared, that is done
+    looked up by key or keys are compared by the dictionary, that is done
     case-insensitively. The case-insensitivity is defined by performing the
     lookup or comparison on the result of the ``lower()`` method on the key
     value. Therefore, objects used as keys must support the ``lower()`` method.
     If a key object does not do that, :exc:`py:TypeError` is raised.
 
+    The dictionary is case-preserving: Whenever keys are returned, they have
+    the lexical case that was originally specified when adding or updating the
+    item.
+
     The dictionary is ordered: The dictionary maintains the order in which items
     were added for all Python versions supported by this package. This is
     consistent with the ordering behavior of the built-in :class:`py:dict`
     class starting with Python 3.7.
 
-    The dictionary is case-preserving: Whenever key values are returned, they
-    have the lexical case that was originally specified when adding or updating
-    the item.
-
     The :class:`~nocasedict.NocaseDict` class is derived from the abstract base
-    class :class:`py:MutableMapping`, and not from the :class:`py:dict` class.
-    Reason is the unique implementation of :class:`~nocasedict.NocaseDict`,
-    which maintains a single dictionary with the lower-cased keys, and values
-    that are a tuple (original key, value). This supports key based lookup
-    with a single dictionary lookup.
+    class :class:`py:collections.abc.MutableMapping`, and not from the
+    :class:`py:dict` class. Reason is the unique implementation of
```

Comment:
Change reason to because if you make it once sentence or "this is because" if two sentences.

Reply:
DONE - made one sentence.